src/ocaml-index/tests/tests-dirs/index-project.t

```diff
@@ -132,3 +132,39 @@
   - 0 compilation units shapes
   - root dir: none
   
+
+Jane Street Merlin uses cms files instead of cmt files. Verify that the results using
+cms files are consistent with using cmt files:
```

Comment:
Not totally consistent, and (after discussion) we don't really know why. Which seems fine, but probably this text should say that.

Reply:
done